sdk/python/feast/infra/utils/snowflake_utils.py

```diff
@@ -273,6 +275,175 @@ def create_temporary_sfc_stage(cursor: SnowflakeCursor) -> str:
     return stage_name
 
 
+def write_pandas_binary(
```

Comment:
this looks like it's also similarly copied heavily from the python connector? What do you need that's custom?

Reply:
it converts our entity key and value columns to a binary format